lacework/readme.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Vulnerability scanning with Lacework
+description: Audit your project for security vulnerabilities using Lacework.
+author: Buildkite
+use_cases: ["Security", "CI", "Schedule"]
+tools: ["Lacework"]
+primary_emojis: [:lacework:]
+---
+
+# Scan using Lacework
+
+This template gives you a continuous integration (CI) pipeline that scans a a sample project for security vulnerabilities using Lacework.
+
+At a glance:
+
+- Uses [Lacework](https://www.lacework.com/) and requires an account and API key for security scanning
+- Requires [Docker](https://docs.docker.com/get-docker/)
+
+## How it works
+
+This template:
+
+1. Sets up the environment with Lacework
+2. Runs Lacework over yourproject for security vulnerabilities.
+
+## Next steps
+
+After you select **Use template**, you’ll:
+
+1. Connect your git repository.
+2. Modify the template commands, environment variables, secrets as needed for your project.
+3. [Create](https://docs.lacework.net/console/api-access-keys) and store your Lacework API token securely on your agents.
```

Comment:
Perhaps it's worth calling out the environment variable names required for the [plugin](https://github.com/buildkite-plugins/lacework-buildkite-plugin) so that it's obvious which specifc env is needed? ie. `LW_API_SECRET` `LW_API_KEY`.

We do something similar for other templates 

e.g. https://buildkite.com/pipelines/templates/cd/vercel-deploy?queryID=34d9534528a9a7066ae721c0257dd0c6